badge-maker/lib/color.spec.js

```diff
@@ -18,15 +18,50 @@ test(isHexColor, () => {
 })
 
 test(normalizeColor, () => {
+  // Shields named color.
   given('red').expect('red')
+  given('green').expect('green')
   given('blue').expect('blue')
-  given('4c1').expect('#4c1')
-  given('f00f00').expect('#f00f00')
```

Comment:
Was this removed because it was duplicative of the `ABC123` one?

Reply:
Yes, exactly.